bitnami/elasticsearch/7/debian-12/rootfs/opt/bitnami/scripts/libelasticsearch.sh

```diff
@@ -909,6 +909,9 @@ elasticsearch_healthcheck() {
     local host
 
     host=$(get_elasticsearch_hostname)
+    if validate_ipv6 "$host"; then
+        host="[$host]"
```

Comment:
Just a little suggestion. As good practice, we usually use `${var}` when calling variables inside strings, to make the code clearer and avoid substitutions or expansion errors . 

```suggestion
        host="[${host}]"
```